crankers/src/bin/main.rs

```diff
@@ -0,0 +1,138 @@
+use std::{collections::HashMap, path::PathBuf, time::Duration};
+
+use anyhow::{anyhow, Context};
+use clap::{arg, Parser};
+use jito_bytemuck::AccountDeserialize;
+use jito_vault_core::{vault::Vault, vault_operator_delegation::VaultOperatorDelegation};
+use jito_vault_cranker::{restaking_handler::RestakingHandler, vault_handler::VaultHandler};
+use log::info;
+use solana_rpc_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::{pubkey::Pubkey, signature::read_keypair_file};
+
+#[derive(Parser)]
+struct Args {
+    /// RPC URL for the cluster
+    #[arg(short, long, env, default_value = "https://api.devnet.solana.com")]
+    rpc_url: String,
+
+    /// Path to keypair used to pay
+    #[arg(short, long, env = "KEYPAIR_PATH")]
+    keypair: PathBuf,
+
+    /// Vault program ID (Pubkey as base58 string)
+    #[arg(
+        long,
+        env,
+        default_value = "Vau1t6sLNxnzB7ZDsef8TLbPLfyZMYXH8WTNqUdm9g8"
+    )]
+    vault_program_id: Pubkey,
+
+    /// Restaking program ID (Pubkey as base58 string)
+    #[arg(
+        long,
+        env,
+        default_value = "RestkWeAVL8fRGgzhfeoqFhsqKRchg6aa1XrcH96z4Q"
+    )]
+    restaking_program_id: Pubkey,
+
+    /// Interval in seconds between cranking attempts (default: 5 minutes)
+    #[arg(long, env, default_value = "300")]
```

Comment:
nit: you can use default_vault_t on thes without string (wont work for pubkey i think)